config_ssm.go

```diff
@@ -0,0 +1,111 @@
+package evergreen
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"github.com/pkg/errors"
+)
+
+var (
+	awsConfig                    *aws.Config
+	ssmParameterPath, ssmEnabled = os.LookupEnv("SSM_PARAMETER_PATH")
+	ssmDisabledErr               = errors.New("SSM is not enabled")
+)
+
+func getClient(ctx context.Context) (*ssm.Client, error) {
```

Comment:
Ty for already writing the boilerplate for me to move project vars to Parameter Store 😎 (no change)